runtime/builtin_types_test.go

```diff
@@ -23,6 +23,33 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
+		fooObject := newTestClass("Foo", []*Type{ObjectType}, toDictUnsafe(args[0])).ToObject()
```

Comment:
I just found out actual tests in TestObjectDelAttr in object_test.go. It does not seems to dynamically  SetAttr first for each tests, so may be I will follow that way.